pipeline/outputs/oci-logging-analytics.md

```diff
@@ -0,0 +1,187 @@
+---
+description: Send logs to OCI Logging Analytics Service
+---
+
+# OCI Logging Analytics
```

Comment:
We do not introduce the acronym `OCI` until later, personally I think we should be clear this is Oracle Cloud Infrastructure Logging and nothing to do with the Open Container Initiative which is what I would expect of that acronym.

Can you update the description and title to use the full name?

Reply:
Fixed